documentation/src/main/wiki/FAQ.md

```diff
@@ -0,0 +1,144 @@
+- [3. User interface and reports](#3-user-interface-and-reports)
+  - [3.1 What browser should I use to browse AET reports?](#31-what-browser-should-i-use-to-browse-aet-reports)
+  - [3.2 I would like to compare pages between two environments/domains. How can I do that with the AET Tool?](#32-i-would-like-to-compare-pages-between-two-environmentsdomains-how-can-i-do-that-with-the-aet-tool)
+  - [3.3 How can I access pages that require authentication to enter?](#33-how-can-i-access-pages-that-require-authentication-to-enter)
+  - [3.4 How can I simulate another user agent?](#34-how-can-i-simulate-another-user-agent)
+  - [3.5 What can I do with elements that render differently each time page is open and it occurs in changes detected each time?](#35-what-can-i-do-with-elements-that-render-differently-each-time-page-is-open-and-it-occurs-in-changes-detected-each-time)
+  - [3.6 I want to check how my page behaves on mobile devices, how can I do that?](#36-i-want-to-check-how-my-page-behaves-on-mobile-devices-how-can-i-do-that)
+- [4. Best practices](#4-best-practices)
+  - [4.1 I want to check how my page looks on several devices - what is the best way do configure suite?](#41-i-want-to-check-how-my-page-looks-on-several-devices---what-is-the-best-way-do-configure-suite)
+
+## 1. Setup and first steps
+
+### 1.1. What do I need to star using AET?
+You need Maven and Java installed to run AET and Chrome browser to see reports if you have AET environment
+already set up. See more information in [[Running Suite|RunningSuite#requirements]] section.
+To setup AET you need [VirtualBox 5.1.14](https://www.virtualbox.org/wiki/Downloads) to run AET VM.
+See more details in [[Basic Setup|BasicSetup]] section.
+
+
+### 1.2. I have AET environment, how can I run tests?
+You need Maven and Java installed to run AET and Chrome browser to see reports. 
+See more information in [[Running Suite|RunningSuite#requirements]] section.
+To start AET suite you need 2 XML files:
+- [[pom.xml|RunningSuite#pomxml]] that defines how to run [[AET Client Application|ClientApplication]],
+- [[suite.xml|DefiningSuite]] that defines [[AET suite|SuiteStructure]].
+
+To run suite simply execute this command: 
+
+`mvn aet:run`
+
+from the directory where you have two above XML files.
+
+Next, you will see [[processing progress|TrackingProgress]]. After a 
+[[SUCCESSFUL BUILD|TrackingProgress#when-tests-successfully-finish---command-line]]
+explore `target` directory and open `redirect.html` in Chrome browser file to see the [[report|SuiteReport]].
+
+
+### 1.3. How can I integrate AET with continuous integration tools?
+AET tests are triggered by [[AET Maven Client Application|ClientApplication]]. Thanks to this you may simply
+run `mvn aet:run` from your favourite CI tool (e.g. [Jenkins](https://jenkins.io/) or 
+[Bamboo](https://www.atlassian.com/software/bamboo)).
+
+### 1.4. I run AET for the first time, how the tool will know patterns?
```

Comment:
How about: _I'm running AET for the first time. How will the tool know the patterns?_